oidc/oidc.go

```diff
@@ -131,7 +125,7 @@ func NewProvider(ctx context.Context, issuer string) (*Provider, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer resp.Body.Close()
+	defer func() { _ = resp.Body.Close() }()
```

Comment:
nit: Never seen this one? Also seems like it's not what the standard library does?

https://github.com/golang/go/search?q=%22defer+resp.Body.Close%22

Reply:
Goland complains about this being a potentially unhandled error.  I reverted those changes.